tests/protractor/e2e/forms/submit-delivery-form.specs.js

```diff
@@ -28,6 +87,26 @@ describe('Submit Delivery Report', () => {
       return element(by.css('#report-form')).isPresent();
     }, 10000);
 
-    expect(helper.isTextDisplayed('Delivery'));
+    //select name
+    deliveryReport.selectPatientName('jack');
+    deliveryReport.goNext();
+    helper.waitElementToBeVisisble(element(by.css('[value="healthy"]')));
```

Comment:
The goNext() is actually clicking the 'Next' button which is on multiple pages of the form. So, we cannot include this one since it is only valid for one page. Maybe what I need to research on is if there is such a thing as a generic 'wait for page to load' and use that instead of waiting for a particular element. That would be useful in many instances.